### ISSUE TITLE
PowerShell Preview fails to open after install

### ISSUE THREAD

**Issue by msftrncs** [PowerShell/PowerShell#8442]
Immediately after installation (upgrade from previous 6.2 preview) 6.2.0 preview 3 will not run, the console window just quickly flashes and disappears, or sometimes its so fast nothing appears at all, but I can see the focus move away and then back.

Windows 10 1809 (Insiders)

PowerShell 6.1.1 still works.

I don't see any log files to report.

I have not yet tried removing and reinstalling.

**Comment by kborowinski** [comment]
It's the same issue as with PowerShell 6.2.0 preview 2 (#8289). Run the installer again and use repair function. It will fix the problem.

**Comment by MaximoTrinidad** [comment]
Workaround on https://github.com/PowerShell/PowerShell/issues/8289

**Comment by msftrncs** [comment]
@kborowinski, repair not working …  one time gave 'account already exists', and another time, I have dialog now waiting for me to relocate the missing install package.

So I have a hunch on why this may have affected me.  I installed the MSI package directly from the website without saving the download, so the dialog is looking for the install package in Edge's temporary download folder.

So now I have redownloaded and saved it … (that MSI file wouldn't repair it) … uninstalled it and then reinstalled and all is fine.   

Looking back through things, this is the first time I have installed the preview without saving it first.  However, it appears that I did a straight run of the current released version without any difficulty.
